netbench-cdk/netbench-monitor/handler.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import boto3
+import logging
+from datetime import datetime, timezone
+from os import getenv
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Max instance lifetime in seconds defaulting to one day.
+# Over-rideable via an environment variable.
+MAX_LIFETIME: int = int(getenv("MAX_LIFETIME", 86400))
+
+def get_ebs_age(date_obj: datetime|str) -> int:
+    # Convert a date object or str to age in seconds
+
+    # Date object conversion is done automajically with boto3
+    # but not with a standard json.load() from tests
+    if type(date_obj) == str:
+      date_format = '%Y-%m-%dT%H:%M:%S%z'
+      date_obj = datetime.strptime(date_obj, date_format)
```

Comment:
what happens if it doesnt match?

Reply:
Adding an exception and a test case to bail out if it doesn't match